app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java

```diff
@@ -57,12 +48,9 @@ public static ReportsFragment newInstance() {
 	private TracingViewModel tracingViewModel;
 	private SecureStorage secureStorage;
 
-	private ReportsSlidePageAdapter pagerAdapter;
-
-	private ViewPager2 headerViewPager;
+	private View headerFragmentContainer;
 	private LockableScrollView scrollView;
 	private View scrollViewFirstchild;
-	private CirclePageIndicator circlePageIndicator;
 
 	private View healthyView;
 	private View saveOthersView;
```

Comment:
Just below the variable `originalFirstChildPadding` seems to be unused (but Github won't let me comment on that line since it's too far out of the diff 🙄 )